playbooks/openstack/openshift-cluster/launch.yml

```diff
@@ -128,6 +128,6 @@
       - parsed_outputs.node_floating_ips
       - parsed_outputs.infra_floating_ips
 
-- include: update.yml
+- include: setup.yml
```

Comment:
@akram I see now that you have an update.yml playbook too. What is that for? It says it's to "Launch Instances"